spread/project_test.go

```diff
@@ -62,35 +63,96 @@ type projectSuite struct{}
 
 var _ = Suite(&projectSuite{})
 
+type loadTest struct {
+	filename string // the file name to write the project to
+	reroot   string // the new root relative to the spread.yaml location
+	errMsg   string // the expected error message
+}
+
+var loadTests = []loadTest{
+	{
+		filename: "spread.yaml",
+		reroot:   "",
+		errMsg:   "",
+	},
+	{
```

Comment:
Thanks, the test looks a lot more readable indeed.  Just one more detail: there's a minor twist here that we use in most projects to optimize the use of space, both horizontal and vertical:
```
var loadTests = []loadTest{{
	filename: "spread.yaml",
	reroot:   "",
	errMsg:   "",
}, {
	...
}, {
	...
}}
```
You can actually see that style in use at the bottom of the diff, on the tests for reindent.